src/scenes/AcceleratorRouter/Cohorts/CohortsListView.tsx

```diff
@@ -96,42 +118,46 @@ const CohortsTable = ({ filterModifiers, extraTableFilters }: CohortTableProps)
       fuzzySearchColumns={fuzzySearchColumns}
       id='cohortsTable'
       Renderer={CohortCellRenderer}
+      rightComponent={actionMenus}
       rows={cohorts || []}
+      title={strings.COHORTS}
     />
   );
 };
 
-export default CohortsTable;
+export default CohortsListView;
```

Comment:
Move to end of file? I don't know if there's a preference but having one place for non-implicit exports (either beginning or ending of file). We tend to use the end of file.

Reply:
I copied this pattern from another view somewhere, but tend to agree that's it's good to have one preferred place. Will move.